### PR TITLE
Adjust the vector docs to use langword instead of cref to preserve generic type information where relevant

### PR DESCRIPTION
**Pull request by tannergooding**
This fixes up the docs for https://github.com/dotnet/dotnet-api-docs/pull/10155

### CODE REVIEW

File: src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs [hunk 1]
```diff
@@ -90,102 +90,102 @@ public static Vector<TTo> As<TFrom, TTo>(this Vector<TFrom> vector)
 #endif
         }
 
-        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see cref="Vector{Byte}" />.</summary>
+        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see langword="Vector{Byte}" />.</summary>
```
Comment (gewarren) [inline src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs:93]:
```suggestion
        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see langword="Vector<Byte>" />.</summary>
```

Would angle brackets be better?

**Review by gewarren — APPROVED** [review]
Thank you!

File: src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs (hunk 1 above)
Comment (gewarren) [reply src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs:93]: Or maybe angle brackets aren't allowed.

Comment (carlossanlop) [reply src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs:93]: I think they are allowed but they need to be escaped. Take a look at the API DocIDs.

Comment (tannergooding) [reply src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs]: Fixed.

**Pull request merged**